mrc/localization/offset_probability/probability.py

```diff
@@ -0,0 +1,18 @@
+import math
+
+from mrc.utils.maths import gauss_distribution
+
+
+class PositionPredictor:
+    def __init__(self, sigma, mu, coef):
+        self.mu = mu
+        self.sigma = sigma
+        self.coef = coef
+
+    def __call__(self, rotation_angle):
```

Comment:
Why rotation_angle? It's supposed to be the angle at which we can see another robot.